executor/src/wasm_executor.rs

```diff
@@ -27,7 +27,7 @@ use state_machine::{Externalities, CodeExecutor};
 use error::{Error, ErrorKind, Result};
 use wasm_utils::{MemoryInstance, UserDefinedElements,
 	AddModuleWithoutFullDependentInstance};
-use primitives::{ed25519, blake2_256, twox_128, twox_256};
+use primitives::{blake2_256, twox_128, twox_256};
```

Comment:
why does primitives contain crypto-hashing but not crypto-signing?

Reply:
ideally primitives wouldn't contain either. But moving the `ed25519` stuff out was necessary to get it to compile correctly for wasm.

Reply:
IMO it's beyond the scope of this PR but definitely a welcome refactor